charts/content-store/templates/slack-notify-sync.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: batch/v1
```

Comment:
Why is this in the app Helm charts? Surely it's conceptually part of the CD config so would make more sense in argocd-apps - and then you'd probably be able to avoid copy-pasting is 65+ times 😅 (which I think probably wouldn't be acceptable from a maintenance perspective - future GOV.UK developers would curse us if they ever wanted to change the way the Slack notifications worked!)